Jenkinsfile

```diff
@@ -41,8 +41,11 @@ properties([
         stringParam(defaultValue: '10.40.9.2', description: 'IP of the WAN board',
             name: 'WAN_IP'),
         stringParam(defaultValue: '/home/jenkins/boardfarm/boardfarm',
-            description: 'Path to the a local copy of Boardfarm repository',
-            name: 'BOARDFARM_DIRECTORY'),
+            description: 'Path to a local copy of OTA scripts',
+            name: 'OTA_DIRECTORY'),
+        stringParam(defaultValue: 'https://github.com/CreatorDev/boardfarm.git',
```

Comment:
I agree that boardfarm url can be hardcoded. Also I would prefer if BOARDFARM_BRANCH was called OVERRIDE_BOARDFARM and worked in the same way as the other overrides for consistency reasons. As for the rest of them that could be env vars thats the plan and this was a temporary stepping stone but I guess it we could just as easily move it to the machine now?

One other thing, having to setup a webserver on the machine is a pain. In the past we have solved this by spinning one up on the fly using `python -m SimpleHTTPServer`.